### PR TITLE
Implement cucumber tests

### PR DESCRIPTION
**Pull request by Astashevich**

### CODE REVIEW

File: src/test/java/by/onliner/cucumber/runner/RunCucumberTest.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package by.onliner.cucumber.runner;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+
+@CucumberOptions(
+        features = "src/test/resources/features",
+        glue = "by/onliner/cucumber")
+        //tags = "@foo, @bar")
```
Comment (Anassstasi) [inline src/test/java/by/onliner/cucumber/runner/RunCucumberTest.java]: Please remove redundant comments

File: src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import static by.onliner.utils.EqualsUtil.equalContains;
+
+public class ShoppingCartSteps extends AbstractSteps {
+
+
```
Comment (Anassstasi) [inline src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java]: Remove extra space line

File: src/test/java/by/onliner/cucumber/runner/RunCucumberTest.java (hunk 1 above)
Comment (Astashevich) [reply src/test/java/by/onliner/cucumber/runner/RunCucumberTest.java]: Removed

File: src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    @When("Remove item from the shopping cart")
+    public void removeItemFromCart() {
+        shoppingCartPage.removeItemFromCart();
+    }
+
+    @Then("Appeared information should contain [Вы удалили]")
+    public void checkedRemovedItemInfo() {
+        String removedItemMessage = shoppingCartPage.getRemovedItemInformation();
+        Assert.assertTrue(equalContains(removedItemMessage, "Вы удалили"),
+                String.format("The message [%s] wasn't contains at expected removed message [%s...]",
```
Comment (Anassstasi) [inline src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java:41]: Do you need '...' in the [%s...] statement ?

File: src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java (hunk 2 above)
Comment (Astashevich) [reply src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java]: Fixed

File: src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java (hunk 3 above)
Comment (Astashevich) [reply src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java:41]: Removed

File: src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                        removedItemMessage, "Вы удалили"));
+    }
+
+    @And("Expected empty cart message should contain [Ваша корзина пуста]")
+    public void checkedEmptyCartMessage() {
```
Comment (Anassstasi) [inline src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java]: Make this method reusable, pass the needed message as an argument

File: src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        mainPage.getMenu().openShoppingCartPage();
+    }
+
+    @When("Remove item from the shopping cart")
+    public void removeItemFromCart() {
+        shoppingCartPage.removeItemFromCart();
+    }
+
+    @Then("Appeared information should contain [Вы удалили]")
+    public void checkedRemovedItemInfo() {
```
Comment (Anassstasi) [inline src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java]: Make this method reusable, pass the needed message as an argument

File: src/test/resources/features/shoppingCart.feature [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Feature: Tests for shopping cart
```
Comment (Anassstasi) [inline src/test/resources/features/shoppingCart.feature:2]: Please use tags for the scenario/feature

File: src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java (hunk 4 above)
Comment (Astashevich) [reply src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java]: Fixed

File: src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java (hunk 5 above)
Comment (Astashevich) [reply src/test/java/by/onliner/cucumber/steps/ShoppingCartSteps.java]: Fixed

File: src/test/resources/features/shoppingCart.feature (hunk 6 above)
Comment (Astashevich) [reply src/test/resources/features/shoppingCart.feature:2]: Used

**Review by Anassstasi — APPROVED** [review]
LGTM!

**Pull request merged**